automation/taskcluster/lib/tasks.py

```diff
@@ -470,21 +470,77 @@ def craft_upload_apk_nimbledroid_task(self, assemble_task_id):
             dependencies=[assemble_task_id],
         )
 
-    def craft_raptor_task(self, signing_task_id, mozharness_task_id, variant):
+    def craft_raptor_speedometer_task(self, signing_task_id, mozharness_task_id, variant, gecko_revision):
+        return self._craft_raptor_task(
+            signing_task_id,
+            mozharness_task_id,
+            variant,
+            gecko_revision,
+            name_prefix='raptor speedometer',
+            description='Raptor Speedometer on the Reference Browser',
+            test_name='raptor-speedometer',
+            job_symbol='sp',
+        )
+
+    def craft_raptor_speedometer_power_task(self, signing_task_id, mozharness_task_id, variant, gecko_revision):
+        return self._craft_raptor_task(
+            signing_task_id,
+            mozharness_task_id,
+            variant,
+            gecko_revision,
+            name_prefix='raptor speedometer power',
+            description='Raptor Speedometer power on the Reference Browser',
+            test_name='raptor-speedometer',
+            job_symbol='sp',
+            group_symbol='Rap-P',
+            extra_test_args=[
+                "--power-test",
+                "--page-cycles 5",
+                "--host HOST_IP",
```

Comment:
Which system defined the (environment variable?) called `HOST_IP`?

Reply:
I confess I don't know. I blindly copied what's run on mozilla-central https://tools.taskcluster.net/groups/U06k1CCySBCEzXL1UhQRjQ/tasks/OeVpShacTB2qdiLwrl_W5w/details

Reply:
HOST_IP comes from raptor:
https://searchfox.org/mozilla-central/source/testing/raptor/raptor/cmdline.py#146

we don't know the IP until we have the host we are running on.

you can also see this defined in m-c: https://searchfox.org/mozilla-central/source/taskcluster/ci/test/raptor.yml#658